sos/report/plugins/vmware.py

```diff
@@ -6,10 +6,10 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, RedHatPlugin
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
```

Comment:
Unless I'm mistaken, I think you can add IndependentPlugin and remove RedHatPlugin to cover both distro families.

Reply:
Don't worry, most of us make a mess every now and then (and some of us pretty often :D)
If you need a hand, you can find us in libera irc network. If I'm online I'll be more than happy to help

Reply:
Thanks for the offer @jcastill ! My colleague @arif-ali got me sorted and I just did a force-push to change it to `IndependentPlugin` :slightly_smiling_face:

Reply:
@arif-ali++ 
it looks perfect now, thank you!